apps/payments/next/app/[locale]/[offeringId]/[interval]/location/page.tsx

```diff
@@ -33,8 +36,17 @@ export default async function Location({
   Sentry.captureMessage('Could not locate user by their ip');
 
   let cms: PageContentOfferingTransformed | undefined;
+  let locationStatus: LocationStatus | undefined;
   try {
     cms = await fetchCMSData(params.offeringId, acceptLanguage, params.locale);
+    const { status } = await determineLocation(
```

Comment:
nit. `const { status: locationStatus } = await determineLocation(` lets you get rid of line 49.

nit. async calls on lines 41 and 42..48, can be added into a `Promise.all()`

Reply:
1st nit is no longer applicable. 2nd nit looks good. 👍 Thank you for the change.